src/templates/ad/index.html.twig

```diff
@@ -0,0 +1,39 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Ad index{% endblock %}
+
+{% block body %}
+    <h1>Ad index</h1>
+
+    <table class="table">
+        <thead>
+            <tr>
+                <th>Id</th>
```

Comment:
Don't need to display  id !

Reply:
Sure, that just the base file generated by the `make:crud` command, gonna disappeared in front PR I guess ? :)

Reply:
Yeah bv